dev/NavigationView/NavigationView.cpp

```diff
@@ -248,6 +250,15 @@ void NavigationView::OnApplyTemplate()
         m_leftNavListViewItemClickRevoker = leftNavListView.ItemClick(winrt::auto_revoke, { this, &NavigationView::OnItemClick });
 
         SetNavigationViewListPosition(leftNavListView, NavigationViewListPosition::LeftNav);
+
+        // Set up the ViewModel for the List
+        auto leftNavigationViewList = leftNavListView.try_as<NavigationViewList>();
+        leftNavigationViewList->ListViewModel(winrt::make_self<ViewModel>());
+        auto viewModel = leftNavigationViewList->ListViewModel();
+        viewModel->IsContentMode(true);
+        viewModel->PrepareView(m_rootNode.get());
+        viewModel->SetOwningList(leftNavListView);
+        leftNavigationViewList->ItemsSource(*viewModel.get());
```

Comment:
>ItemsSource [](start = 32, length = 11)

Please do a testing, start with left nav, then switch to topnav, then switch back. I expect a crash, and we should not regress on this part.

Reply:
Verified that there is no crash. The ListView has its own viewmodel, which is why I don't set that to null on pane mode switch, however the viewmodel internally clears the ItemsSource vector which ensures that we don't have the same items in two different places.